### PR TITLE
Added KrakenD link

### PR DESCRIPTION
**Pull request by diroussel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,6 +234,7 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 
 * [Kong](https://github.com/Kong/kong) - Scalable, distributed, and plugin oriented API gateway backed by Nginx.
 * [Tyk API Gateway](https://github.com/TykTechnologies/tyk) - Lightweight API gateway with analytics logging, written in Go.
+* [KrakenD](https://github.com/devopsfaith/krakend) Ultra performant API Gateway with middleware. Written in Go.
```
Comment (fzaninotto) [inline README.md]: Please add it to the end of the list

File: README.md [hunk 2]
```diff
@@ -220,6 +220,7 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [Marvel Comics API](http://developer.marvel.com/) - Query characters, stories, events about Marvel superheroes.
 * [JSON Placeholder](http://jsonplaceholder.typicode.com/) - Free online REST service that you can use whenever you need some fake data.
 * [APIs.guru](http://APIs.guru) - Wikipedia for Web APIs, each API has OpenAPI/Swagger description.
+* [Postman Echo Service](https://docs.postman-echo.com/?version=latest) - Postman Echo is service you can use to test your REST clients and make sample API calls. It provides endpoints for GET, POST, PUT, various auth mechanisms and other utility endpoints.
```
Comment (fzaninotto) [inline README.md]: I went to this site to try and understand what it's useful for, and I couldn't. Please remove it.

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (diroussel) [reply README.md]: Ok, will do.  I put it next to Tyk, as it's more similar to Tyk.  But will move down as requested.

File: README.md (hunk 2 above)
Comment (diroussel) [reply README.md]: ok

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**